File: college/python-data-structure/fatorial.py
# fatorial de n

def fatorial(n):
    if(n>1):
        return n*fatorial(n-1)
    else:
        return n
    
def fatorial_uma_linha(n):
    return (n*fatorial_uma_linha(n-1)) if n > 1 else 1

x = 5
print(fatorial(x))
print(fatorial_uma_linha(x))

File: college/cpp-intro/fatorial.cpp
#include <iostream>

using namespace std;

int main(){
	int num,i,fat;
	cout<<"digite um numero: ";
	cin>>num;
	fat=num;
	for(i=num-1;i>0;i--){
		fat = fat*i;
	}
	cout<<"O fatorial de num e: "<<fat;
}
